### PR TITLE
feat: Update Signup screen

### PR DESCRIPTION
**Pull request by pranavrajs**
Fixes https://github.com/chatwoot/marketing/issues/59

Screenshot 2022-11-30 at 8 37 22 PM

- Testimonials are dynamic, they will be fetched from testimonials.cdn.chatwoot.com/content.json URL.

### CODE REVIEW

File: app/javascript/dashboard/routes/auth/components/Testimonials/Index.vue [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            :author-name="testimonial.authorName"
+            :author-designation="testimonial.authorCompany"
+            :class="`testimonial-${index ? 'right' : 'left'}--card`"
+          />
+        </div>
+      </div>
+    </div>
+    <testimonial-footer
+      title="All it takes is one step to move forward."
+      sub-title="You're one step away from engaging your customers, retaining them and finding new ones."
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/auth/components/Testimonials/Index.vue:26]: i18n?

File: app/javascript/dashboard/routes/auth/components/Signup/Form.vue [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+      this.$refs.hCaptcha.reset();
+      this.credentials.hCaptchaClientResponse = '';
+      this.didCaptchaReset = true;
+    },
+  },
+};
+</script>
+<style scoped lang="scss">
+.h-captcha--box {
+  margin-bottom: var(--space-one);
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/auth/components/Signup/Form.vue:195]:
```suggestion
  margin-bottom: var(--space-small);
```

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Comment by nithindavid** [comment]
image
Should we use the company name instead of the Account name? We could also use a company icon too

**Comment by pranavrajs** [comment]
1. Made the update to the company name 
Screenshot 2022-12-07 at 3 42 02 PM


2. I couldn't get the text working properly. So I have removed it for now.

File: app/views/installation/onboarding/index.html.erb [hunk 3]
```diff
@@ -31,7 +31,7 @@
                 </label>
                 <label>
                   <span>Company Name</span>
-                  <%= text_field :user, :company, placeholder: "Enter an account name. eg: Wayne Enterprises", required: true %>
+                  <%= text_field :user, :company, placeholder: "Enter a company name. eg: Wayne Enterprises", required: true %>
```
Comment (pranavrajs) [inline app/views/installation/onboarding/index.html.erb:34]:
```suggestion
                  <%= text_field :user, :company, placeholder: "Enter your company name. eg: Wayne Enterprises", required: true %>
```

**Pull request merged**